eval.go

```diff
@@ -33,7 +33,7 @@ import (
 const maxUInt16 = 0xffff
 
 func (m *FileMatch) addScore(what string, s float64, debugScore bool) {
-	if debugScore {
+	if s != 0 && debugScore {
```

Comment:
Not sure about this. s==0 means that we actually found a symbol kind but didn't give it a score, no? This seems useful to show because it points to gap in our scoring. How else would we figure out such a gap just by looking at the debug output?

Reply:
See my other or making us default to a factor of 1 for unknown symbols